sdk/typescript/test/e2e/signature-mpc.test.ts

```diff
@@ -142,3 +144,13 @@ describe('Test key share transfer', () => {
 		).toBeTruthy();
 	});
 });
+
+describe('Derive a paillier keypair from a Sui keypair', () => {
+	it('should derive a paillier keypair from a Sui keypair', async () => {
+		let keypair = Ed25519Keypair.generate();
+		const [encryptionKey, decryptionKey] = generate_keypair_from_bytes(
+			new TextEncoder().encode(keypair.export().privateKey),
+		);
```

Comment:
This should be an exported TS function that gets `keypair` and returns the Paillier key.